nexus/db-queries/src/db/datastore/role.rs

```diff
@@ -209,6 +209,11 @@ impl DataStore {
         // We might instead want to first-class the idea of Policies in the
         // database so that we can build up a whole new Policy in batches and
         // then flip the resource over to using it.
+
+        // This method should probably be retryable, but this is slightly
```

Comment:
Might be good to start an issue listing these kinds of queries, which we want to put inside the retry wrapper, but can't do easily as they're written?

Reply:
Filed https://github.com/oxidecomputer/omicron/issues/7213